src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -291,10 +293,10 @@ private X509CertificateContainer CreateCertificate(bool isAuthority, bool isMach
                 var authorityKeyIdentifier = new AuthorityKeyIdentifier(
                     SubjectPublicKeyInfoFactory.CreateSubjectPublicKeyInfo(_authorityKeyPair.Public),
                     new GeneralNames(new GeneralName(authorityX509Name)),
-                    new BigInteger(7, _random).Abs());
+                    serialNum);
 
-                _certGenerator.AddExtension(X509Extensions.AuthorityKeyIdentifier, true, authorityKeyIdentifier);
-                _certGenerator.AddExtension(X509Extensions.KeyUsage, true, new KeyUsage(X509KeyUsage.DigitalSignature | X509KeyUsage.KeyAgreement | X509KeyUsage.KeyCertSign | X509KeyUsage.KeyEncipherment | X509KeyUsage.CrlSign));
+                _certGenerator.AddExtension(X509Extensions.AuthorityKeyIdentifier, false, authorityKeyIdentifier);
```

Comment:
Nit: not required for merge, but it would be better for readability if the parameters all used their names.  To find out what the change from 'true' to 'false' meant, I had to go look at code not in our repo (BouncyCastle).